src/praisonai-agents/praisonaiagents/memory/enhanced_memory.py

```
@@ -0,0 +1,647 @@
+                conn.commit()
+                conn.close()
+                return True
+        except Exception as e:
+            logger.error(f"Legacy delete failed for key {key}: {e}")
+        return False
+    
+    # -------------------------------------------------------------------------
+    #            Wrapper Methods for Backward Compatibility
+    # -------------------------------------------------------------------------
+    
+    def store_short_term(self, text: str, metadata: Dict[str, Any] = None, **kwargs):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)